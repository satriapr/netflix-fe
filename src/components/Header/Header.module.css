.logo {
  float: left;
  font-size: 30px;
}

.logo a {
  color: #cc0000;
  text-decoration: none;
}

.logo a:hover {
  color: #e60000;
}

.searchBar {
  float: right;
}

.clear {
  clear: both
}

@media only screen and (max-width: 690px) {
  .logo,
  .searchBar {
    float: none;
  }

  .logo {
    text-align: center;
  }
}