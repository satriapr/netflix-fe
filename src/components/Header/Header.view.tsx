import React from 'react'
import { Link, withRouter } from 'react-router-dom'

import Constant from '../../constants/Constant'
import Search from '../Search/Search.view'
import styles from './Header.module.css'

const Header = () => {
  return (
    <header>
      <nav>
        <div className={styles.logo}>
          <Link to="/">{Constant.NETFLIX}</Link>
        </div>
        <div className={styles.searchBar}>
          <Search />
        </div>
        <div className={styles.clear} />
      </nav>
    </header>
  )
}

export default withRouter(Header)
