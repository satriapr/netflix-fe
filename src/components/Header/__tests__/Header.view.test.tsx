import { testSnapshots } from '../../../utils/test.util'
import Header from '../Header.view'

describe('Header Component', () => {
  testSnapshots(Header, [
    {
      description: 'Header props',
      props: {}
    }
  ])
})
