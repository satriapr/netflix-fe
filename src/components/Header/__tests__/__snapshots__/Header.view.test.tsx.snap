// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component snapshots Header props 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
